src/app.rs

```diff
@@ -68,46 +86,69 @@ impl DialsApp {
 
         style.visuals = egui::style::Visuals::dark();
 
+        for (_text_style, font_id) in style.text_styles.iter_mut() {
+            font_id.size = 72.0;
+        }
+
         cc.egui_ctx.set_style(style);
     }
 
     pub fn ui(&mut self, ctx: &egui::Context) {
-        self.dial_ui(ctx);
-        self.tracking_ui(ctx);
+        let state = self.state_mutex.lock().unwrap();
+
+        match state.deref() {
```

Comment:
nit: using the deref operator is more idiomatic over the trait imo. Gjengset uses `&*` and `&mut *` exclusively to invoke deref. Ex: https://github.com/jonhoo/rust-imap/blob/main/src/types/acls.rs#L165

This is akin to doing `let a = Index::index(vec, 1)` or:
```Rust
use std::ops::Deref
let a = vec.index(1);
```
instead of simply: `let a = vec[1];`

Reply:
Good point, I've switched it all over